kotsu/run.py

```diff
@@ -33,23 +34,21 @@ def run(
             each model through.
         results_path: The file path to which the results will be written to, and results from prior
             runs will be read from.
-        skip_if_prior_result: Flag, if True then will not run validation-model combinations
-            that are found in the results at given results_path. If False then all combinations
-            will be ran and any prior results in results_path will be completely overwritten.
+        force_rerun: Specify if "all" models should be re-ran and their previous results
```

Comment:
typo: `re-ran` -> `re-run` as it's future tense

Reply:
bb09356